### ISSUE TITLE
Notification icon cause error: "mage path "%s" must be absolute"

### ISSUE THREAD

**Issue by jelomada** [heimrichhannot/contao-pwa-bundle#9]
Hi!
There is an error when trying to send notification with icon in contao 4.9.4.
this: https://github.com/heimrichhannot/contao-pwa-bundle/blob/9d6b65937ae1ceb10f89a9c64b64301bed320d84/src/Notification/DefaultNotification.php#L125 cause error from https://github.com/contao/core-bundle/blob/77f9f7adb0b5bdef81e2b8afd170e410714c8e71/src/Image/ImageFactory.php#L135

```php
            if (!$this->filesystem->isAbsolutePath($path)) {
                throw new \InvalidArgumentException(sprintf('Image path "%s" must be absolute', $path));
            }
```

Could be fixed by:

This line:
```php
$image = System::getContainer()->get('contao.image.image_factory')->create($file->path, $iconSize);
```

should be something like this:
```php
$image = System::getContainer()->get('contao.image.image_factory')->create(System::getContainer()->getParameter('kernel.project_dir') . '/' . $file->path, $iconSize);
```

Even better with checking if file exists:

```php
	public function setIconFromModel(string $icon, string $iconSize = null)
	{
		if ($iconSize)
		{
			$iconSize = deserialize($iconSize);
		}        
                       
        $rootDir = System::getContainer()->getParameter('kernel.project_dir');
        $imageFactory = System::getContainer()->get('contao.image.image_factory');
        if (
            ($objModel = FilesModel::findByUuid($icon)) != null &&
            is_file($rootDir . '/' . $objModel->path)
        ) {
            $image = $imageFactory->create($rootDir . '/' . $objModel->path, $iconSize);
            $this->setIcon($image->getUrl($rootDir));
        }
	}
```

**Comment by koertho** [comment]
@jelomada Thanks for reporting. Could you do a pull request for this?

**Comment by jelomada** [comment]
@koertho Yes. I can try but this will be my first time, so be gentle, please :)

**Comment by koertho** [comment]
@jelomada Your fix is now available in version 0.7.3
